### PR TITLE
Fix +mixin style include results in "TypeError: Cannot read property 'first' of undefined"

### PR DESCRIPTION
**Pull request by dotdat**
DCO 1.1 Signed-off-by: Brian George <bpgeorge88@gmail.com>

Sorry it took me so long to get this back into review.

### CODE REVIEW

**Comment by benthemonkey** [comment]
Is there a GitHub issue that this fixes? If so, can you include a link to it?

**Comment by dotdat** [comment]
Sure thing: https://github.com/sasstools/sass-lint/issues/380

**Comment by benthemonkey** [comment]
LGTM but I'll wait for someone else to look too.

File: tests/sass/extends-before-declarations.sass [hunk 1]
```diff
@@ -27,3 +27,8 @@
   .bar
     @extend %waldo
     content: 'where'
+
+
+.foo-noextend
+  +foo
+  content: "bar"
```
Comment (DanPurdy) [inline tests/sass/extends-before-declarations.sass]: Missing new line at end of file

File: tests/sass/extends-before-declarations.scss [hunk 2]
```diff
@@ -29,3 +29,7 @@
     content: 'where';
   }
 }
+
+.foo-noextend {
+  @include foo;
+}
```
Comment (DanPurdy) [inline tests/sass/extends-before-declarations.scss]: Missing new line at end of file

**Comment by DanPurdy** [comment]
Hi @dotdat no problem, thank you for taking the time to fix this issue. Two extremely minor issues but would be good to add them in as it's defined in our editorconfig that we require newlines at the ends of files. Do you use an IDE with editorconfig support?

This is good to merge once it's updated. Thanks again!

**Comment by dotdat** [comment]
Hey @DanPurdy -- sorry for the massive delay, the last month left me with 0 free time. I've updated this according to the editorconfig specs and just wanted to double check back before I did anything. Thanks!

**Comment by DanPurdy** [comment]
Hi @dotdat, no problem I think most of us have been in the same boat recently! Thanks for taking the time here looks good. 

On a side note you may just want to add the email address that you just used to do those last commits as a secondary email for your github account to get the proper contribution numbers.

:smile:

**Pull request merged**